components/style/themes/default.less

```diff
@@ -290,6 +290,10 @@
 @table-padding-vertical: 16px;
 @table-padding-horizontal: 8px;
 
+// Tag
+// --
+@tag-default-bg: @background-color-base;
```

Comment:
do you want me to make `@tag-default-bg` #f3f3f3 or create a different variable?